index.js

```diff
@@ -12,6 +12,11 @@ export default {
     if (options.action && options.action.color) {
       options.action.color = processColor(options.action.color);
     }
+
+     if(!options.tintColor) {
```

Comment:
Could we call this `textColor` instead? And it should probably also be added to the docs.